lightning/src/ln/channel.rs

```diff
@@ -6615,29 +6615,31 @@ impl<SP: Deref> OutboundV1Channel<SP> where SP::Target: SignerProvider {
 		let keys = self.context.get_holder_pubkeys();
 
 		msgs::OpenChannel {
-			chain_hash,
-			temporary_channel_id: self.context.channel_id,
-			funding_satoshis: self.context.channel_value_satoshis,
+			common_fields: msgs::CommonOpenChannelFields {
+				chain_hash,
+				temporary_channel_id: self.context.channel_id,
+				funding_satoshis: self.context.channel_value_satoshis,
+				dust_limit_satoshis: self.context.holder_dust_limit_satoshis,
+				max_htlc_value_in_flight_msat: self.context.holder_max_htlc_value_in_flight_msat,
+				htlc_minimum_msat: self.context.holder_htlc_minimum_msat,
+				commitment_feerate_sat_per_1000_weight: self.context.feerate_per_kw as u32,
```

Comment:
I am curious as to why chose to name it so verbosely.

Maybe a simplification?
```suggestion
				commitment_feerate_per_kw: self.context.feerate_per_kw as u32,
```

Reply:
At the time the accept_channel2/open_channel2 messages were introduced, there was rough consensus on using the longer form going forward. So the common struct here just uses that longer form. I wanted to avoid making unrelated changes here, but open to comment.

Reply:
Completely reasonable point!

If the consensus is towards the longer naming, it's completely fine by me.
But if we have scope for a little tweak, I would personally prefer a more concise name that maintains brevity while ensuring adequate description.